# This service is used to delete an existing target from the interoperability
# server.

# Target ID to delete.
uint64 id

---

# Whether the request was successful.
bool success
